registry/cloudflare-workers/serverless.yml

```diff
@@ -20,17 +20,28 @@ inputTypes:
                 required: true
 
 outputTypes:
-        success:
+        successScript:
                 type: boolean
                 description: If worker script has been successfully deployed
-        result:
+        resultScript:
                 type: any
                 description: The resultant worker script uploaded
-        errors:
+        errorsScript:
                 type: any
                 description: A list of error messages
-        messages:
+        messagesScript:
+                type: any
+                description: Messages
+        successRoute:
+                type: boolean
+                description: If worker script has been successfully deployed
+        resultRoute:
+                type: any
+                description: The resultant worker script uploaded
```

Comment:
This is the same description as `resultScript`, I'm not sure what these do from reading the descriptions

Reply:
I'll take care of the copy. Will open another PR to my branch after we merge this